src/main/java/com/example/sixthmafiabot/repository/GameRepository.java

```diff
@@ -1,24 +1,51 @@
 package com.example.sixthmafiabot.repository;
 
-
+import com.example.sixthmafiabot.DTO.CreateGameDTO;
+import com.example.sixthmafiabot.DTO.UpdateGameDTO;
 import com.example.sixthmafiabot.models.Environment;
 import com.example.sixthmafiabot.models.Game;
-import org.springframework.data.repository.Repository;
-import org.springframework.scheduling.annotation.Async;
+import com.example.sixthmafiabot.repository.Abstract.BaseRepository;
+import com.example.sixthmafiabot.repository.Abstract.SpringRepositoryImplementations.SpringGameRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class GameRepository extends BaseRepository {
+
+    @Autowired
+    SpringGameRepository springGameRepository;
+
+    public Game create(CreateGameDTO createGameDTO) {
+
+        Game game = modelMapper.map(createGameDTO, Game.class);
+
+        springGameRepository.save(game);
+
+        return game;
+    }
+
+    public Game update(Game gameInDatabase, UpdateGameDTO updateGameDTO){
+
+        gameInDatabase.setGameStatus(updateGameDTO.getGameStatus());
```

Comment:
validate status change
e.g. game in status FINISHED can not change status to STARTED